WebLinking.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = 'WebLinking'
-  spec.version = '1.1.0'
+  spec.version = '3.0.0'
```

Comment:
Sure. I just think it makes sense to bump the major version, so nobody accidentally updates to this.